### DISCUSSION TITLE
Error train-test splitting

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#359] [Q&A] [answered]
Saya sedang melakukan forecasting pada data kategorik menggunakan model RNN. Pada saat menentukan minmaxscalernya terjadi error karena tipe data saya category. 

Datanya : 
<img width="373" alt="image" src="https://user-images.githubusercontent.com/106010284/192731208-2868c898-94ae-4be0-a67f-baf3cf0a62f0.png">

Kodenya : 
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler, MinMaxScaler
df_train, df_test = train_test_split(df_attack, test_size=24*7, shuffle=False)
scaler = MinMaxScaler()
scaled_train = scaler.fit_transform(df_train)
scaled_test = scaler.transform(df_test)

Errornya : 
ValueError: could not convert string to float: 'CA'

Solusinya bagaimana ya ?  Apakah saya harus memberi label datanya terlebih dahulu ?
Terima kasih

**Comment by jafarhf1** [comment, marked as answer]
Terkait kendala yang bapak hadapi, sebagaimana yang sudah sempat dijelaskan data yang bapak gunakan lebih sesuai untuk kasus klasifikasi karena bapak ingin memprediksi kolom kategorik. Selain itu, fungsi scaling hanya digunakan untuk tipe data yang numerik
